macosx/mac-app.py

```diff
@@ -45,7 +45,7 @@
   help = 'force execution even if SCRIPT does not exist')
 parser.add_argument('-v', '--version', \
   help = 'version string for the application bundle, \
-  visible e.g. in \'Get Info\' and in \'Open with...\'', default = appinfo.version)
+  visible e.g. in \'Get Info\' and in \'Open with...\'', default = appinfo.running_version())
```

Comment:
Please don't change this: see the comment about macosx/build-dmg.sh.
Here it is only the default value, and when build-dmg.sh passes the version explicitly to mac-app.py, but I don't see any advantage in changing this default.